patient-management/patient-management-impl/src/main/java/ksch/patientmanagement/patient/PatientNumberIndex.java

```diff
@@ -17,7 +17,7 @@
 @AllArgsConstructor
 @Getter
 @Setter
-@TableGenerator(name="patient_index_number", initialValue = 999)
+@TableGenerator(name = "patient_index_number", initialValue = 999)
```

Comment:
Something looked wrong in the database with the tables for the medical record ids. Please have a look at the generated tables.